packages/harpguru-core/src/components/harp-guru/harp-guru.tsx

```diff
@@ -1,75 +1,84 @@
 import 'react-native-gesture-handler'
 
-import { PanGestureHandler } from 'react-native-gesture-handler'
-import { View, StyleSheet } from 'react-native'
+import { createProvider } from 'reactn'
+import { withTimingTransition, useValue } from 'react-native-redash'
+import Animated, { Easing, interpolate } from 'react-native-reanimated'
+import { Dimensions, StyleSheet } from 'react-native'
 import React from 'react'
 import type { ReactElement } from 'react'
 
-import { QuizQuestionDisplay } from '../quiz-question-display'
-import { LayoutMenu } from '../layout-menu'
-import { HarpFaceMemo } from '../harp-face'
-import { CovariantMenu } from '../covariant-menu'
-import { ActivityLegend } from '../activity-legend'
-import { setGlobalState } from '../../utils'
-import { MenuStates, MenuStashPosition } from '../../types'
-import { colors } from '../../styles'
-import { getSizes } from '../../styles'
+import { HarpGuruPage } from '../harp-guru-page'
 
-import { useFlushBufferedActivityToggles, useMenus } from './hooks'
+import { getInitialGlobalState } from './utils'
 
-setGlobalState()
-
-const styles = StyleSheet.create({
-  fillScreen: {
-    ...StyleSheet.absoluteFillObject,
-    backgroundColor: colors.pageColor,
-    justifyContent: 'center',
-    alignItems: 'center',
-  },
-})
+const initialGlobalState = getInitialGlobalState()
+const Provider1 = createProvider(initialGlobalState)
+const Provider2 = createProvider(initialGlobalState)
+const Provider3 = createProvider(initialGlobalState)
 
 export const HarpGuru = (): ReactElement => {
-  const [menuState, handleSwipe, handleTap] = useMenus()
-  const covariantOpenCloseTapHandler = () => {
-    handleTap(MenuStates.CovariantMenu)
-  }
-  const layoutOpenCloseTapHandler = () => {
-    handleTap(MenuStates.LayoutMenu)
-  }
+  const pageInFrame = useValue<0 | 1 | 2>(0)
 
-  const sizes = getSizes()
-  const { 8: swipeThreshold } = sizes
+  const { width: windowWidth, height: windowHeight } = Dimensions.get('window')
+  const deviceShortSide =
+    windowWidth < windowHeight ? windowWidth : windowHeight
 
-  useFlushBufferedActivityToggles()
+  const pageTransition = withTimingTransition(pageInFrame, {
+    duration: 300,
```

Comment:
There is an animation duration value somewhere we could recycle here.

Reply:
The animation duration for the menu's is also defined as a magic number. Both of them need to be changed.

Reply:
Recorded as techdebt in https://github.com/js-jslog/harpguru/issues/52#issuecomment-737527652

Reply:
Moved to separate issue #54